src/entity/Player.ts

```diff
@@ -234,7 +239,6 @@ export function create(clientId: string, underworld: Underworld): IPlayer {
   player.unit.healthMax = PLAYER_BASE_HEALTH;
 
   underworld.players.push(player);
-  updateGlobalRefToCurrentClientPlayer(player, underworld);
```

Comment:
Why is this no longer needed on Player.create?

Reply:
Hmm... Removing this line fixed an issue that caused an incorrect assignment of globalThis.player when starting a hotseat game, and I left it at that because removing this line didn't seem to cause any other bugs. Player.Load() seems to set globalThis.player before any issues arise.

In theory it still makes sense to immediately assign globalThis.player after player creation, and doing so may future-proof the game/prevent future bugs, so I've made another commit that ensures globalThis.player is assigned immediately after player creation, for the primary client player.